github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/config.groovy

```diff
@@ -39,8 +39,10 @@ f.block {
             )
         }
 
-        f.optionalProperty(title: "User Restriction", field: "userRestriction")
-   
-        f.optionalProperty(title: "Branch Restriction", field: "branchRestriction")
+        f.advanced() {
```

Comment:
Frankly speaking :-1: for having huge advanced uncollapsible button instead of 2 small checkboxes.
Would prefer introduce `experimental` feature toggle in global config because this features are not fully tested/implemented.